src/Http/Client.php

```diff
@@ -61,6 +61,8 @@ public function __construct($dsn)
      */
     public function sendCommand(CommandInterface $command)
     {
+        $request = clone $this->request;
```

Comment:
Without this clone call, the client cannot safely be used multiple times with different commands